test/signin.test.js

```diff
@@ -1,53 +1,34 @@
-const request = require('supertest');
-const app = require('../app');
-const { defaultUser } = require('./constants');
+const { createUser, authenticateUser } = require('./utils');
+const { factory } = require('factory-girl');
+
+let user = null;
+
+beforeAll(async () => {
+  const userFactoryGirlResponse = await factory.build('User');
```

Comment:
you should use `create` instead of  `build`. The create() method persists the instance of the model while the build() method keeps it only on memory.